### PR TITLE
Add option to encrypt the root partition in cgdisk-manual-install

### PR DESCRIPTION
**Pull request by mvincerx**

### CODE REVIEW

File: source/clear-linux/get-started/cgdisk-manual-install/cgdisk-manual-install.rst [hunk 1]
```diff
@@ -265,21 +265,40 @@ partitions and select to format them.
...
+      `Set mount points` now show as completed. 
+   
    .. figure:: figures/cgdisk-manual-install-15.png
       :scale: 50 %
-      :alt: Set mount point completed
+      :alt: Encrypt root partition
+
+      Figure 15: :guilabel:`Encrypt root partition`
+
+#. Type a confirmation passphrase as directed.
```
Comment (tpleavitt) [inline source/clear-linux/get-started/cgdisk-manual-install/cgdisk-manual-install.rst]: Question on this:  Are there any rules that a user is required to follow for creating a passphrase?

Comment (mrkz) [reply source/clear-linux/get-started/cgdisk-manual-install/cgdisk-manual-install.rst]: oh, I apologize, I completely missed this, I'm taking a look now

Comment (mrkz) [reply source/clear-linux/get-started/cgdisk-manual-install/cgdisk-manual-install.rst]: @tpleavitt there is not rules for creating the passphrase

**Comment by tpleavitt** [comment]
Thanks @mrkz , looks good to me to

**Pull request merged**